src/__tests__/App.test.js

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from "@testing-library/react";
+import App from "../components/App";
+
+describe("App", () => {
+  it("Renders Correctly", () => {
+    const { asFragment } = render(<App />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("Displays NASA Logo", () => {
+    render(<App />);
+    const logoElement = screen.getByAltText("Nasa Logo");
+    expect(logoElement).toBeInTheDocument();
+    expect(logoElement.src).toContain(
+      "https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
+    );
+  });
+
+  it("Displays Search Component", () => {
+    render(<App />);
+    const searchElement = screen.getByTestId("search-component");
+    expect(searchElement).toBeInTheDocument();
+  });
```

Comment:
It's best to avoid using test id's in your test unless there is no other option. To check that the `searchElement` is rendered, you check check for something that is rendered as part of `Search`, e.g. an input with the placeholder text `"Enter keywords..."` or the button with the text `"Go!"`